app/assets/javascripts/assignment_form.js

```diff
@@ -7,12 +7,13 @@ function importOptions(starterCodeFieldValue) {
   }
 }
 
-function removeErrorBox() {
-  errorBoxes = document.getElementsByClassName("error");
-  for (var i = 0; i < errorBoxes.length; i++) {
-    errorBox = errorBoxes[i]
-    if (errorBox.innerText == "Starter code repository is not a template repository. Make it a template repository to use template cloning.") {
-      errorBox.style.display = 'none';
-    }
-  }
+function removeErrorBox(el) {
+  parent = el.closest(".errored");
```

Comment:
I think we need a keyword in front of this: `const`, `let` or `var`. The code you're replacing didn't have them, but we should start adding them in to eliminate unneeded globals:

```suggestion
  var parent = el.closest(".errored");
```